Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/CarbonBlackEnterpriseEDR.py

```diff
@@ -1197,6 +1270,9 @@ def get_file_path_command(client: Client, args: dict) -> CommandResults:
 
 
 def fetch_incidents(client: Client, fetch_time: str, fetch_limit: str, last_run: dict) -> tuple[list, dict]:
+    # The new API version (v7) always returns one duplicate alert, except for the first run.
+    if last_run:
+        fetch_limit = 1 + arg_to_number(fetch_limit)  # type: ignore - we are sending 50 if user doesn't choose a limit.
```

Comment:
Why not start with parsing the fetch limit so there will be no need for type: ignore?

Reply:
if not limit := arg_to_number(fetch_limit):
    raise ValueError ("limit cannot be empty.")